source/connect/advanced-connection-options/advanced-connection.txt

```diff
@@ -78,6 +78,14 @@ Procedure
                read preference option. 
              - Any specified tag sets. 
 
+      The following screenshot shows the :guilabel:`Advanced`
+      tab configured with the read preference of :guilabel:`Secondary
+      Preferred`. 
```

Comment:
It might help to add a bit of context here explaining why a user might want to use "secondary preferred".